docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -1717,8 +1749,890 @@ Here are the steps:
 
 If you see an error, then consult [Debugging]({{< relref "/userguide/managing-domains/model-in-image/debugging.md" >}}) in the Model in Image user guide.
 
-This completes the sample scenarios.
+If you plan to run the [Update3 use case](#update3-use-case), then leave your domain running.
+
+### Update2 use case
+
+This use case demonstrates concurrently deploying a domain that is similar to the [Update1 use case](#update1-use-case) domain to the same `sample-domain1-ns` namespace but with a different domain UID, a different WebLogic domain name, and a different WebLogic domain encryption key. It does this by:
+
+- Using the same image, image model YAML, and application archive as the Initial and Update1 use cases.
+- Using the same model update ConfigMap source file as the Update1 use case (a data source).
+- Using a different (unique) domain UID, `sample-domain2`, for the new domain.
+- Using a different (unique) domain name, `domain2`, for the different domains.
+- Deploying secrets and a model update ConfigMap that are uniquely labeled and named for the new domain.
+
+Note that this use case shows Model in Image's unique ability to quickly deploy a copy of a WebLogic domain that has a different WebLogic domain name and domain encryption key. This is a useful capability that is not supported by the Domain in Image domain home source type:
+
+- A Domain in Image domain home source type does not support overriding the domain name, but different domain names are necessary when two domains need to interoperate. In this use case, we take advantage of model macros to ensure that its two different domains have a different domain name: 
+
+  - First, we define the domain name in the model YAML using the `@@ENV:CUSTOM_DOMAIN_NAME@@` environment variable macro.
+  - Second, we set the value of the `CUSTOM_DOMAIN_NAME` environment variable differently using the `env` stanza in each domain's domain resource.
+
+- Domain in Image requires that its images embed a WebLogic `security/SerializedSystemIni.dat` domain encryption key that cannot be changed for the image (see [why layering matters]({{< relref "/userguide/cicd/why-layering-matters.md" >}}) in CI/CD Considerations). This necessarily means that two Domain in Image domains that share the same image have the ability to decrypt each other's encrypted passwords. A Model in Image's domain encryption key, on the other hand, is not embedded in the image and is instead dynamically and uniquely created each time the domain is started. 
```

Comment:
and is instead dynamically -> and instead, is dynamically

Reply:
done